_develop/connect/use-with-clarity.md

```diff
@@ -87,40 +124,104 @@ const functionArguments = [
 
 If you're using Typescript, these Clarity types can be imported as `ContractCallArgumentType` from `@blockstack/connect`.
```

Comment:
maybe a link to the class/file would be good?

Reply:
I like the suggestion, but right now we don't have a stable branch that this code lives on yet. Typescript developers will most likely have IDE autocomplete, so importing this enum will show them the options automatically.